sys/include/net/sock/util.h

```diff
@@ -49,6 +50,7 @@ int sock_udp_ep_fmt(const sock_udp_ep_t *endpoint, char *addr_str, uint16_t *por
  * @note Caller has to make sure hostport and urlpath can hold the results!
  *       Make sure to provide space for SOCK_HOSTPORT_MAXLEN respectively
  *       SOCK_URLPATH_MAXLEN bytes.
```

Comment:
Just noticed that those are not references in rendered doc... If you want to, you can add those in a separate commit, otherwise, let's do this in a separate PR.

Reply:
I'm cleaning up anyways, added them to the next fixup